ssz/sedes/serializable.py

```diff
@@ -416,6 +338,35 @@ def __new__(cls, name, bases, attrs):
             ),
         )
 
+    #
+    # Implement BaseSedes methods as pass-throughs to the container sedes
+    #
+    def serialize(cls: Type[TSerializable], value: TSerializable) -> bytes:
```

Comment:
Are these intended to be `classmethod` decorated?  If not, I'm confused as to what is going on here.

Reply:
Those are methods defined in the metaclass, so they become classmethods automatically.